### ISSUE TITLE
Salt Ag Grid Theme Missing Text Truncation

### ISSUE THREAD

**Issue by origami-z** [jpmorganchase/salt-ds#1766]
### Package name(s)

AG Grid Theme (@salt-ds/ag-grid-theme)

### Package version(s)

v1.1.4

### Description

Text truncation doesn't work in **Salt** AG grid theme, but works in **UITK** version. This is likely due to `display: flex` applied on the cell.

This mostly impacts on right aligned text (mostly numbers) where most significant digits are getting lost.

Report from MTK team.

### Steps to reproduce

Use demo at https://storybook.saltdesignsystem.com/?path=/docs/documentation-data-grid-ag-grid-theme--page
Shrink any column, no text gets automatically truncated.

### Expected behavior

Text should be truncated when column too narrow, just like UITK theme.

### Operating system

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [X] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.

**Comment by james-nash** [comment]
Bug ranking score: **59** (using rubric v3)

Priority: **high**

```
========== BEGIN BUG RUBRIC SCORE DATA ===========
eyJ2YWx1ZXMiOnsiYWZmZWN0ZWRQZW9wbGVPcHRpb25zIjoyNS
wiYWZmZWN0ZWRDb21wb25lbnRzT3B0aW9ucyI6NywiZnJlcXVl
bmN5T3B0aW9ucyI6MTksImltcGFjdE9wdGlvbnMiOjIwLCJhZm
ZlY3RlZERzT3B0aW9ucyI6MTcsInVyZ2VuY3lPcHRpb25zIjox
OSwieEZhY3Rvck9wdGlvbnMiOjMwfSwicnVicmljVmVyc2lvbi
I6M30=
=========== END BUG RUBRIC SCORE DATA ============
```

**Comment by ZarkoPernar** [comment]
@origami-z You are correct! I don't know why I did not expect that behaviour. 😅 

Pr to fix this #1778
